integreat_cms/cms/templates/events/event_form_sidebar/venue_box.html

```diff
@@ -65,5 +66,13 @@
                 {% translate "Open on Google Maps" %}
             </a>
         </div>
+        <div id="poi-ajax-success-message"
+             class="w-full flex flex-wrap justify-center items-center bg-green-200 font-bold hidden">
+            {% trans "The new location was successfully created." %}
+        </div>
+        <div id="poi-ajax-error-message"
+             class="w-full flex flex-wrap justify-center items-center bg-red-400 font-bold hidden">
```

Comment:
```suggestion
             class="bg-red-100 border-l-4 border-red-500 text-red-500 px-4 py-3 hidden">
```
I would style this the same way as our other error messages